spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-spi/src/main/java/org/springframework/cloud/data/module/deployer/ModuleLauncherArgumentsHelper.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.data.module.deployer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A helper class to generate qualified environmental properties from a
```

Comment:
I'm not a fan of using "Helper" in the classname since it's not descriptive and often redundant. For example, could we change this to `ModuleLauncherArgumentQualifier`? (note: singular for "Argument")

On that note, should we consider adding a `getQualifiedArgs()` method to ModuleDeploymentRequest *instead* of introducing this class?